utils/runtime-fuzzer/src/runtime/mod.rs

```diff
@@ -91,13 +96,17 @@ pub fn new_test_ext() -> TestExternalities {
     ext
 }
 
-pub fn increase_to_max_balance(who: AccountId) -> DispatchResultWithPostInfo {
-    let new_reserved = BalancesPallet::<Runtime>::reserved_balance(&who);
+pub fn set_account_balance(account: AccountId, balance: Balance) -> DispatchResultWithPostInfo {
```

Comment:
don't you want to avoid hacks in favour of real-looking-execution and just call Balances::transfer from some huge pre-funded account?

Reply:
_fuzzer related comments_
resolving, a new PR for fuzzer changes will be opened.